docs/issues_catched.md

```diff
@@ -0,0 +1,34 @@
+
+# Issues Precaution finds
+
+Precaution utilizes multiple security static code analysis tools.
+That is the reason why the amount and types of checks for the different
+programming languages differs.
+The security static code analyzers are language-focused
+and some of the vulnerabilities they find are specific 
+for the programming language.
+However, some of those rules can be grouped:
+
+* Hardcoded credentials
+* SQL injections
+* Command injections
+* HTML injections
+* Weak random number generator
+* Imported and used weak cryptographic algorithms
+* Imported insecure libraries
+* Use of specific insecure functions from libraries
+* Weak key used for a cryptographic algorithm
+* Eval function used
+* Binding to all network interfaces example when using IP "0.0.0.0"
+* TLS turned off or bad versions used
+* Bad file permissions set during file creation
+* Poor file permissions used with chmod
+
+**Note:** Those vulnerabilities aims to warn the developers to audit a specific piece of code.
```

Comment:
Perhaps something like:

```suggestion
**Note:** Precaution (through the underlying tools) warns developers of specific sections of code that may require security review.
```

Reply:
Yes, I don’t know why I wrote it the way I did :D. It’s confusing.